mobile/lib/providers/image/immich_local_thumbnail_provider.dart

```diff
@@ -82,11 +82,14 @@ class ImmichLocalThumbnailProvider
 
   @override
   bool operator ==(Object other) {
-    if (other is! ImmichLocalThumbnailProvider) return false;
     if (identical(this, other)) return true;
-    return asset == other.asset;
+    if (other is ImmichLocalThumbnailProvider) {
```

Comment:
Code changes here are bigger than strictly necessary, but I made the implementation consistent with `immich_local_image_provider.dart`